### DISCUSSION TITLE
Feedback: don't jump to front of screen on localhost reload

### DISCUSSION THREAD

**Discussion by mmicheux** [responsively-org/responsively-app#967] [General]
THIS APP IS GREAT! I love it and it's made my developing life much easier, so thank you!! 

One piece of feedback. When I am developing on localhost and there is a change, when the site reloads, the app tends to jump forward to be at the top of the view on my screen. Usually I just need it to reload in the background, not be the top program visible on my monitor. Hope this makes sense!

**Comment by manojVivek** [comment]
Thank you so much for the kind words!

This has been reported a few times, we tried a fix and looks like it didn't resolve the problem completely. We'll check this out.

**Comment by manojVivek** [comment]
@mmicheux PS: If your employer has some kind of FOSS Contributor Fund, I would appreciate it if you could use that to support us: https://responsively.app/sponsor/ :)
